docs/docs/reference/release-notes/image-migration-guide.md

```diff
@@ -0,0 +1,234 @@
+  )
+}
+```
+
+### API Changes
+
+In addition to the syntax changes for using gatsby-plugin-image, there are also changes to the API that affect the resolver arguments (and StaticImage).
+
+#### fluid
+
+The `fluid` image type has been deprecated in favor of two alternatives.
+
+The first is an image type called `fullWidth`. This image is designed to be used for things like hero images and generates image sizes accordingly. Instead of passing something like `maxWidth`, it takes an array called `breakpoints` that will generate images designed for those screen sizes. Note that `fullWidth` images will expand to fit the width of their container, even if that width is larger than the source image.
+
+The second is a responsive image type called `constrained` that generates smaller images, but nothing larger than the original image source size. Additionally, you can pass `width` and/or `height` to limit the largest image generated to that size.
+
+#### maxWidth
+
+`maxWidth` and `maxHeight` are deprecated for all image types. For constrained and fixed images, use `width` and `height`. For fullWidth images look at the `breakpoints` option.
+
+#### aspectRatio
+
+`aspectRatio` is a new argument that takes a number (or fraction). If you pass it without also passing in `width` or `height`, it will default to using the source image width, and then adjusting the height to the correct aspect ratio. Alternatively you can pass your own width or height and it will set the other dimension. Passing both `height` and `width` will cause `aspectRatio` to be ignored in favor of the inferred aspect ratio.
+
+#### formats
+
+Previously, images generated their own type, e.g. jpg, png, etc. You could also generate webp images when using the appropriate fragment. This is now controlled using the `formats` argument. This field takes an array, `["AUTO", "WEBP"]` by default, `"AVIF"` is also valid.
+
+#### options nested inside objects
+
+Previously, transformations like `grayscale` and quality options such as `pngQuality` were top level query arguments. This has changed.
+
+See the [how to guide] for specifics, but `grayscale` now exists within the `transformOptions` argument, and `pngQuality` inside `pngOptions`.
+
+### Breaking changes
+
+Due to the changes to gatsby-plugin-image, there is some functionality that is no longer supported.
+
+1. GatsbyImage is no longer a class component and therefore cannot be extended
+2. Fluid images no longer exist, and the fullWidth replacement does not take maxWidth or maxHeight
+3. Art direction is no longer supported
+4. The component no longer takes a decomposed object and the following code is not valid
+
+```javascript
+// THIS IS NOT VALID
+<GatsbyImage image={{ src: example.src, srcSet: ``, width: 100 }} />
+```
+
+## How to Migrate
+
+1. Install and update dependencies
+
+   ```shell
+   npm install gatsby-plugin-image gatsby-plugin-sharp gatsby-transformer-sharp
```

Comment:
```suggestion
   npm install gatsby-plugin-image gatsby-plugin-sharp gatsby-transformer-sharp gatsby-source-filesystem
```

Reply:
I didn't add this because they are migrating from gatsby-image and therefore won't be using StaticImage unless they opt to manually. filesystem isn't necessary for GatsbyImage usage unless I'm missing something?